app/assets/stylesheets/reset.scss

```diff
@@ -123,6 +123,10 @@ input, textarea, button, select {
   vertical-align: baseline;
   font-family: inherit; }
 
+select {
```

Comment:
Check the first comment here https://github.com/sharetribe/sharetribe/pull/2052 and the explanation about `select` tags. I hope it's not necessary, but I don't know how to circumvent it.